source/js/weekly_navigation.js

```diff
@@ -63,30 +180,117 @@ function populate(key) {
     // Sets all dates for each day
     const days = document.getElementsByClassName('date');
     for (let i = 0; i < 7; i += 1) {
+
+        // loop through every log
         const tempKey = getDaysKey(window.curDate);
+        const log = getDaysData(tempKey);
+        const allTasks = log.tasks;
+        window.curDate.setDate(window.curDate.getDate() + 1);
+        
+        if(test_events[tempKey] == undefined){ 
+            continue;
+        }
+        console.log(tempKey + ", " + test_events[tempKey].name);
```

Comment:
Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
        console.log(`${tempKey  }, ${  test_events[tempKey].name}`);
```